src/persist/src/client.rs

```diff
@@ -275,24 +306,31 @@ impl<K: Codec, V: Codec> StreamWriteHandle<K, V> {
 
     /// Unblocks compaction for updates at or before `since`.
     ///
-    /// The compaction frontier can never decrease and it is an error to call this function with a
-    /// since argument that is less than the current compaction frontier.
+    /// The compaction frontier can never decrease and it is an error to call
```

Comment:
nit: it looks like we have editor settings fighting here

Reply:
I'd be all for auto-formatting this as well!

Reply:
lol I didn't know that - i'm happy to auto format this too!

Reply:
reverted